charts/node/values.yaml

```diff
@@ -38,8 +38,8 @@ initContainer:
 ##
 kubectl:
   image:
-    repository: bitnami/kubectl
-    tag: latest
+    repository: paritytech/kubetools
+    tag: kubectl
```

Comment:
Can we have multiple versioned tags? E.g., `paritytech/kubetools:kubectl-1.22`, `paritytech/kubetools:kubectl-latest`

Reply:
`kubectl-latest` is `kubectl`, while specific version is `kubectl-$KUBE_VERSION`.